### PR TITLE
fixed issue with vue@2.2.1

### PR DESCRIPTION
**Pull request by galenyuan**
- migrate build env to webpack2
- fix the issue in vue-popup from vue@2.2.1
- use `export` in entry instead of `module.exports`

Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow the contributing guide.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ if (typeof window !== 'undefined' && window.Vue) {
   install(window.Vue);
 };
 
-module.exports = {
+export {
   install,
```
Comment (QingWei-Li) [inline src/index.js]: s 改成了 es module 是个 break change is大佬可以看看之前一个 issue

File: package.json [hunk 2]
```diff
@@ -39,42 +39,45 @@
   "dependencies": {
     "array-find-index": "^1.0.2",
     "raf.js": "0.0.4",
-    "vue-lazyload": "^0.8.0"
+    "vue-lazyload": "^0.8.3"
   },
   "peerDependencies": {
-    "vue": "^2.1.8"
+    "vue": "^2.2.1"
```
Comment (QingWei-Li) [inline package.json]: 新版不兼容 2.1 了吗

File: src/index.js (hunk 1 above)
Comment (galenyuan) [inline src/index.js]: 换了个思路，export一个带install的对象，同时`export default install`

Comment (QingWei-Li) [inline src/index.js]: 没用的啊。。。你看看我前面那个 PR 就知道了 😂

**Pull request merged**